httpdocs/api/proposals.php

```diff
@@ -102,6 +82,22 @@ function parameter($parameter, $type='') {
   $proposals[] = $proposal;
 }
 $result->free();
+
+$query = "SELECT "
+        ."COUNT(*) AS number_of_proposals "
+        ."FROM proposal "
+        ."LEFT JOIN publication ON publication.id = proposal.id "
+        ."LEFT JOIN publication AS area_p ON proposal.area = area_p.signature "
+        ."LEFT JOIN area ON area.id = area_p.id "
+        ."WHERE $secret$open$search"
+        ."YEAR(FROM_UNIXTIME(proposal.deadline)) = $year "
+        ."AND ST_Intersects(area.polygons, ST_Buffer(POINT($longitude, $latitude), $radius))";

```

Comment:
We should avoid a copy/paste of a large chunk of this request and store it in a variable to re-use it.